project.html

```diff
@@ -0,0 +1,512 @@
+        </symbol>
+    </svg>
+    <div class="reviews-container">
+    <h1 class="reviews-title">Guest reviews</h1>
+    <h1 class="reviews-title-adapt">Reviews</h1>
+    <div class="reviews-gallery">
+        <div class="reviews-box">
+            <a href="#">
+            <div class="reviews-cost">From 250 USD</div>
+            <img class="reviews-image" src="images/reviews_images/Bali.svg" alt="Bali" width="400" height="400">
+            </a>
+                <div class="reviews-review">
+
+                <div class="reviews-review-1">
+                <div class="reviews-name">
+                    <p class="reviews-hotel-name name _font-weight-bold"><a href="#">Ubud Bali Resort&SPA</a></p>
+                    <p class="reviews-country-name name"><a href="#">Bali, Indonesia</a></p>
+                </div>
+                <div class="reviews-rating-viewing">
+                   <div id="rectangle">9,4</div>
+                    <p class="reviews-viewing"><a href="#">1324 reviews</a></p>
+                </div>
+                </div>
+
+                <div class="reviews-review-2">
+                    <svg class="reviews-account">
+                        <use href="#reviews-account" />
+                    </svg>
+                    <div class="reviews-user">
+                        <p class="reviews-person">Hanna Ivanova</p>
+                        <img class="reviews-person-country" src="images/reviews_images/ukraine_flag.svg" alt="Ukraine" width="83" height="21">
+                    </div>
+                </div>
+
+                <div class="reviews-feedback">
+                    <p>Great location, really pleasant and clean rooms, but the thing that makes this such a good place to stay are the staff. All of the people are incredibly helpful and generous with their time and advice.</p>
+                </div>
+                <div class="reviews-switchers">
+                    <img class="reviews-switcher switcher-1" src="images/reviews_images/switcher_1.svg">
+                    <img class="reviews-switcher switcher-2" src="images/reviews_images/switcher_2.svg">
+                    <img class="reviews-switcher switcher-2" src="images/reviews_images/switcher_2.svg">
+                    <img class="reviews-switcher switcher-2" src="images/reviews_images/switcher_2.svg">
+                </div>
+
+            </div>
+
+
+        </div>
+        <div class="reviews-box">
+            <a href="#">
+            <div class="reviews-cost">From 40 USD</div>
```

Comment:
When photo is not loaded, block with cost randomly displays under the description text
![image](https://user-images.githubusercontent.com/36200303/113488301-d8b2d980-94c5-11eb-90be-80b25cf66792.png)
If with photos it looks good, than OK